scripts/lib/api/conversionPipeline.ts

```diff
@@ -79,27 +114,53 @@ export async function runConversionPipeline(
     let url = `/${relative(`${getRoot()}/docs`, dir)}/${name}`;
     results.push({ ...result, url });
   }
+  return results;
+}
 
-  const allImages = uniqBy(
-    results.flatMap((result) => result.images),
-    (image) => image.fileName,
-  );
-
+async function createNecessaryDirs(
+  results: HtmlToMdResultWithUrl[],
+): Promise<void> {
   const dirsNeeded = uniq(
     results.map((result) => parse(urlToPath(result.url)).dir),
   );
   for (const dir of dirsNeeded) {
     await mkdirp(dir);
   }
+}
+
+async function copyImages(
+  pkg: Pkg,
+  htmlPath: string,
+  results: HtmlToMdResultWithUrl[],
+): Promise<void> {
+  // Some historical versions don't have the `_images` folder in the artifact store in Box (https://ibm.ent.box.com/folder/246867452622)
+  if (pkg.isHistorical() && !(await pathExists(`${htmlPath}/_images`))) return;
```

Comment:
Early returning here is new. We used to have this conditional after `const allImages` and before `await saveImages`. No need to compute `allImages` first.